packages/paste-core/components/combobox/__tests__/index.spec.tsx

```diff
@@ -192,7 +195,7 @@ describe('Combobox', () => {
       render(<ComboboxMock />);
       const renderedLabel = document.querySelector('label');
       const renderedTextbox = screen.getByRole('textbox');
-      expect(renderedLabel.getAttribute('for')).toEqual(renderedTextbox.getAttribute('id'));
+      expect(renderedLabel!.getAttribute('for')).toEqual(renderedTextbox.getAttribute('id'));
```

Comment:
It's for null checks. `renderedTextbox` _might_ be null according to the typing of `getByRole`. So rather than do a null check to can use the `!` to basically say "I know this is here". It's called a non-null assertion https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

Reply:
I'm being lazy in this instance, because I know there is a textbox on the page for this test